recaf-ui/src/main/java/me/coley/recaf/ui/plugin/DownloadFailedException.java

```diff
@@ -0,0 +1,24 @@
+package me.coley.recaf.ui.plugin;
+
+public class DownloadFailedException extends RuntimeException {
+
+    public DownloadFailedException() {
+        super();
+    }
+
+    public DownloadFailedException(String message) {
+        super(message);
+    }
+
+    public DownloadFailedException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public DownloadFailedException(Throwable cause) {
+        super(cause);
+    }
+
+    protected DownloadFailedException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
+        super(message, cause, enableSuppression, writableStackTrace);
+    }
```

Comment:
Don't think this constructor will ever be used. Nor the default one. `message` + `message, cause` are probably the only ones that will be used.

Reply:
There is indeed a problem, I did not focus on this, I will correct it next time